go/vt/sqlparser/parse_test.go

```diff
@@ -6504,6 +6569,112 @@ func TestValidSelectCases(t *testing.T) {
 	testFile(t, "select_cases.txt", makeTestOutput(t))
 }
 
+// TestParseMultiple concatenates all the valid SQL test cases and check it can read
+// them as one long string.
+func TestParseMultiple(t *testing.T) {
```

Comment:
nice!

Reply:
Thank you ❤️

Reply:
Ignore my comment https://github.com/vitessio/vitess/pull/17946/files#r2020560297
I see the tests here now

Reply:
just a thought. This is already a big test file